Add a detailed README and clean up the configuration file (#94)
elastic/elastic-agent-shipper: .github/ISSUE_TEMPLATE/question.md

---
name: Question
about: Who, what, when, where, and how?

---

Hey, stop right there!

We use GitHub to track feature requests and bug reports. Please do not

> submit issues for questions about how to use features of Beat, how to

Reviewer: ```suggestion
submit issues for questions about how to use features of the shipper, how to
```

set Beats up, best practices, or development related help.

> However, we do want to help! Head on over to our official Elastic-Agent-Inputs forums and ask

Reviewer: ```suggestion
However, we do want to help! Head on over to our official Elastic Agent forums and ask
```

your questions there. In additional to awesome, knowledgeable community

> contributors, core Elastic-Agent-Inputs developers are on the forums every single day to help

Reviewer: ```suggestion
contributors, core Elastic Agent developers are on the forums every single day to help
```
